mmdet/core/mask/structures.py

```diff
@@ -0,0 +1,423 @@
+import mmcv
+import numpy as np
+import pycocotools.mask as maskUtils
+import torch
+
+
+class Masks(object):
+
+    def rescale(self, scale, interpolation='nearest'):
+        pass
+
+    def resize(self, h, w, interpolation='nearest'):
+        pass
+
+    def flip(self, flip_direction):
+        pass
+
+    def pad(self, h, w, pad_val):
+        pass
+
+    def crop(self, bbox):
+        pass
+
+    def crop_and_resize(self,
+                        bboxes,
+                        mask_h,
+                        mask_w,
+                        assigned_inds,
+                        interpolation='bilinear'):
+        pass
+
+    def expand(self, expanded_h, expanded_w, top, left):
+        pass
+
+    def to_tensor(self, dtype, device):
+        pass
+
+
+class BitMapMasks(Masks):
+    """This class represents masks in the form of bitmaps.
+
+    Args:
+        masks (ndarray): ndarray of masks in shape (N, H, W), where N is
+            the number of objects.
+        height (int): height of masks
+        width (int): width of masks
+    """
+
+    def __init__(self, masks, height, width):
+        self.height = height
+        self.width = width
```

Comment:
Are `height` and `width` necessary? They can be obtained from `masks` even if the shape of `masks` is (0, h, w).